site/style.css

```diff
@@ -1,11 +1,52 @@
-p {
-	color: red;
+body {
+    background-color: #FAEBD7;
+    font-family:arial;
+    color:black;
+    }
+
+th, td {
+    padding: 20px;
+}
+
+div {
+    font-size:20px;
+}
+
+a {
+    color:black;
+}
+
+ul {
+    padding:10px;
+}
+
+ul li {
+    margin:10px;
+    margin-left:35px;
+    padding:0px;
 }
 
-.greenparagraph {
-	color: blue;
+#header {
+    background-color:#FAEBD7;
+    color:black;
+    clear:both;
+    text-align:center;
+    padding:5px;    
 }
 
-p span {
-	color: #831FB5;
-}
\ No newline at end of file
+#section {
+    background-color:#FAEBD7;
+    color:black;
+    clear:both;
+    text-align:left;
+    padding:5px;    
+}
+
+#footer {
+    background-color:#FAEBD7;
+    color:black;
+    clear:both;
+    text-decoration:none;
+    text-align:center;
+    padding:5px;         
+}
```

Comment:
Great start with styling! I encourage you to add more properties and play around!